### PR TITLE
os/board/rtl8730e: Prepare PM for GPIO

### PR DESCRIPTION
**Pull request by edwakuwaku**
- Prepare PM for GPIO

### CODE REVIEW

File: os/pm/pm_wakehandler.c [hunk 1]
```diff
@@ -55,7 +55,7 @@ void pm_wakehandler(clock_t missing_tick)
 {
 	irqstate_t flags  = enter_critical_section();
 
-	pmllvdbg("missing_tick: %d\n", missing_tick);
+	pmllvdbg("missing_tick: %lld\n", missing_tick);
```
Comment (edwakuwaku) [inline os/pm/pm_wakehandler.c:58]: Hi @ewoodev , the clock_t is using u64, %d is printing wrong value, thus I changed to %lld here.

**Comment by edwakuwaku** [comment]
Hello @ewoodev , @ritesh55555 , other than enabling GPIO as wakeup src, this PR also added informational log to show the wakeup src. For wifi-keepalive, it will also show the packet type received that triggers wakeup.
Please help to try it out~

File: os/pm/pm_wakehandler.c (hunk 1 above)
Comment (ewoodev) [reply os/pm/pm_wakehandler.c:58]: Thank you for change edwakuwaku
this value is usinged long long, so Could you please apply `%llu` ?

Comment (edwakuwaku) [reply os/pm/pm_wakehandler.c]: Hi EunWoo님, thanks for reminding, changed accordingly~

File: build/tools/amebasmart/gnu_utility/km0_km4_app_RELEASE_NOTE.txt [hunk 2]
```diff
@@ -1,5 +1,9 @@
 /* == "version" + "Realtek git version" + "compile date" + "compile time" == */
 
+== version 2eb067f 2024/05/08-12:24:28 ==
+1. Enable GPIO as wakeup interrupt src
```
Comment (ewoodev) [inline build/tools/amebasmart/gnu_utility/km0_km4_app_RELEASE_NOTE.txt:4]: Could you leave the commit comment ?
which GPIO channel enabled?

**Comment by sunghan-chang** [comment]
@edwakuwaku Could you resolve conflicts?

**Comment by sunghan-chang** [comment]
@edwakuwaku Can I merge now?

**Comment by ritesh55555** [comment]
> @edwakuwaku Can I merge now?

Hi @sunghan-chang , i am still testing it with product. Can you give me some more time to confirm?

**Comment by ritesh55555** [comment]
Hi @sunghan-chang @edwakuwaku  , i checked this PR with product code , it is working fine . It can be merged . Board is able to wake up with bixby command

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**